os/drivers/wireless/realtek/os/tizenrt/tizenrt_service.h

```diff
@@ -45,7 +45,7 @@
 #if (defined CONFIG_GSPI_HCI || defined CONFIG_SDIO_HCI) || defined(CONFIG_LX_HCI)
 /* For SPI interface transfer and us delay implementation */
 #if !defined(CONFIG_PLATFORM_8195A) && !defined(CONFIG_PLATFORM_8711B) && !defined(CONFIG_PLATFORM_8721D) && !defined(CONFIG_PLATFORM_8195BHP) && !defined(CONFIG_PLATFORM_8710C) && !defined(CONFIG_PLATFORM_TIZENRT)
-#include <rtwlan_bsp.h>	
+#include <rtwlan_bsp.h>

```

Comment:
Please don't mix different kind of change. Please split changes of code beautifuler.

Reply:
Applied. Thanks for the comment.